build/build.sh

```diff
@@ -261,7 +261,7 @@ fi
 
 # we pick microblaze toolchain from Vitis install
 if [[ -z ${XILINX_VITIS:+x} ]] || [[ ! -d ${XILINX_VITIS} ]]; then
-    export XILINX_VITIS=/proj/xbuilds/2019.2_released/installs/lin64/Vitis/2019.2
+    export XILINX_VITIS=/proj/xbuilds/2023.1_released/installs/lin64/Vitis/HEAD
```

Comment:
```suggestion
    export XILINX_VITIS=/proj/xbuilds/2023.1_released/installs/lin64/Vitis/2023.1
```
Let use the real directory rather than the link.